usr/share/rear/lib/shell-workflow.sh

```diff
@@ -1,13 +1,14 @@
 if [[ "$VERBOSE" ]]; then
     WORKFLOW_shell_DESCRIPTION="start a bash within rear; development tool"
 fi
-WORKFLOWS+=( shell )
-WORKFLOW_shell () {
-    if test "$SIMULATE" ; then
+WORKFLOWS+=(shell)
+WORKFLOW_shell() {
+    if test "$SIMULATE"; then
```

Comment:
please edit the commit to remove this unrelated whitespace change (which is contrary to examples in https://github.com/rear/rear/wiki/Coding-Style as well )